__tests__/app.test.js

```diff
@@ -214,4 +214,31 @@ describe('/api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad request')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('/api/comments/:comment_id', () => {
+    test('DELETE:204 deletes requested comment and does not send response', () => {
+        return request(app).delete('/api/comments/5').expect(204)
+    })
+    test('DELETE:404 responds with an appropriate status and error message when given a non-existent id', () => {
+        return request(app)
+          .delete('/api/comments/99')
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toBe('comment does not exist');
+          });
+      });
+      test('DELETE:400 responds with an appropriate status and error message when given an invalid id', () => {
+        return request(app)
+          .delete('/api/comments/invalid')
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe('Bad request')
+          })
+      })
+      test('DELETE:404 when path is non existent', () => {

```

Comment:
We will likely already have a bad path test, this isn't related to the functionality of this endpoint so shouldn't be in this describe block. Or potentially not needed at all if it auto passed due to already being covered previously.